package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "nodejs-rchain",
+  "version": "0.0.1",
+  "description": "nodejs rchain lib",
+  "main": "index.js",
+  "repository": "https://github.com/zsluedem/nodejs-rchain.git",
+  "author": "Will",
+  "license": "MIT",
```

Comment:
Please also include start scripts to generate JS binding files and hook to do it after install.
```json
"scripts": {
  "rnode-generate": "rnode-grpc --rnode-version v0.9.24",
  "postinstall": "npm run rnode-generate"
}
```

Reply:
I am not acutually sure how to do this in the right way.

Reply:
`npm install` downloads dependencies in `node_modules` and with `postinstall` hook it will also generate `rnode-grpc-gen` folder with `rnode-generate` script command.